ghost/core/test/e2e-server/run-update-check.test.js

```diff
@@ -0,0 +1,54 @@
+const {spawn} = require('child_process');
```

Comment:
We haven't had this type of test in the past where we'd spawn a whole child process etc. I'd go down the path of getting the job running through Ghost-native path - JobsManager. I'm also thinking tests for jobs specifically should get their own directory?

Reply:
Aha, why didn't I think of that 😀 I think i got hung up on the fact `run-update-check.js` is essentially a standalone file that can be executed directly. I've updated the test to make use of the job manager, and also moved the test into its own `jobs` directory 👍